### DISCUSSION TITLE
npx codeceptjs run --steps --grep "\@tag1|\@tag2" doesnt work in powershell but works in cmd

### DISCUSSION THREAD

**Discussion by rkgour7492** [codeceptjs/CodeceptJS#3653] [Q&A] [answered]
i tried all below but no luck. Is this because of some implementation on codeceptjs commands or my limited knowledge of powershell
npx codeceptjs run --steps --grep '\@tag1|\@tag2'
npx codeceptjs run --steps --grep "`@tag1|`@tag2'
npx codeceptjs run --steps --grep 'tag1|tag2'

**Comment by kobenguyent** [comment, marked as answer]
https://github.com/codeceptjs/CodeceptJS/issues/3536

@rkgour7492 You can read more here.

**Reply by rkgour7492** [reply]
It worked like below, thank you 
npx codeceptjs run --steps --grep  "@C123444| @C683248"
